.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
what exactly is the reason for not having package-lock in the repo?

Reply:
there was no package-lock initially, so I didn't pushed it, and there are other arguments for not having package-lock for a lib

Reply:
Thats because I published last time long time ago as npm didn't have lock files.
I thought about removing package-lock and read this block post https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

I think we should keep the lock file and update dependencies to make sure we test with the same versions our users will use.

The downside of loosing stable devDependencies is too big to be.

Reply:
Yes, I don't mind, I'll commit it then. I slightly agreed with this [view](https://github.com/hapijs/contrib/issues/114#issuecomment-340629718) but again I don't mind